tests/common/test_cgroupconfigurator.py

```diff
@@ -130,6 +127,21 @@ def raise_exception(*_):
                     message = args[0]
                     self.assertIn("A TEST EXCEPTION", message)
 
+    def test_get_processes_in_agent_cgroup_should_return_the_processes_within_the_agent_cgroup(self):
```

Comment:
How is this test different from `test_get_processes_in_cgroup_should_return_the_processes_within_the_cgroup` in `test_cgroupapi.py`?

Reply:
code-wise they are the same, but test 2 different components (configurator vs api)